civis/tests/test_futures.py

```diff
@@ -344,7 +347,10 @@ def test_cancel_finished_job():
     fut = ContainerFuture(-10, 100, polling_interval=1, client=c,
                           poll_on_creation=False)
     assert not fut.done()
-    assert fut.cancel() is False
+    with warnings.catch_warnings():
+        # Check that no warnings are raised.
+        warnings.simplefilter("error")
+        assert fut.cancel() is False
```

Comment:
`status_code` of `err_resp` was actually not set properly, thereby raising a warning when `fut.cancel()` was called.